### PR TITLE
Creates CSPM privileges standalone page

### LINKED ISSUE #6261: Update CSPM required permissions
### Description

Per this [comment](https://github.com/elastic/security-team/issues/10681#issuecomment-2442337878) and [issue](https://github.com/elastic/security-team/issues/10957), the required permissions for CSPM getting started guides on ESS 8.16+ and serverless should be explained in more detail. 

There are three possible access levels, each of which have different minimum permissions. The three access levels are Read, Write, and Manage. Because the amount of information we'll be communicating is substantial, I'm leaning towards putting this information on a new page within the [CSPM section ](https://www.elastic.co/guide/en/security/current/cspm.html), and linking to it from the CSPM getting started guides for AWS, GCP, and Azure. 

The required permissions are as follows:

##### ES Index privileges:

-    logs-cloud_security_posture.findings_latest-default (Read)
-    logs-cloud_security_posture.scores-default (Read)

##### Kibana Privileges

-    Security (Read)

### Write permission

Can read cloud posture data, create detection rules from misconfigurations, and enable/disable benchmark rules.

##### ES Index privileges:

 -  logs-cloud_security_posture.findings_latest-default (Read)
 -  logs-cloud_security_posture.scores-default (Read)

##### Kibana Privileges
- Security (All)

###  Manage permission

Can read cloud posture data, create detection rules from misconfigurations, enable/disable benchmark rules and install/update/uninstall CSPM integration and assets.

##### ES Index privileges:

 -   logs-cloud_security_posture.findings_latest-default (Read)
 -   logs-cloud_security_posture.scores-default (Read)

##### Kibana Privileges

-    Security (All)
-    Spaces (All)
-    Fleet (All)
-    Integrations (All)

### Background & resources

* PRs: 
* Issues/metas: 
* Point of contact: Smriti / Paulo
* Test environments: 

### Which documentation set does this change impact?

ESS and serverless

### ESS release

8.16

### Serverless release

current

### Feature differences

-

### API docs impact

-

### Prerequisites, privileges, feature flags

_No response_

### PR DESCRIPTION
**Pull request by benironside**
Fixes #6261 — Creates a standalone page that describes the privileges that grant different levels of access to CSPM features. Removes the lists of privileges from the requirements sections of each of the CSPM get started guides for AWS, GCP, and Azure, and replaces them with a link to the new page, which includes more details.

Preview: [CSPM privilege requirements ESS](https://security-docs_bk_6269.docs-preview.app.elstc.co/guide/en/security/master/cspm-required-permissions.html), [Serverless](https://security-docs_bk_6269.docs-preview.app.elstc.co/guide/en/serverless/main/cspm-required-permissions.html)

### CODE REVIEW

**Review by opauloh — DISMISSED** [review]
Great documentation, thank you @benironside 💯

File: docs/cloud-native-security/cspm-permissions.asciidoc [hunk 1]
```diff
@@ -0,0 +1,61 @@
+[[cspm-required-permissions]]
+= CSPM privilege requirements
+
+This page lists required privilges for {elastic-sec}'s CSPM features. There are three access levels: read, write, and manage. Each access level and its requirements are described below.
```
Comment (natasha-moore-elastic) [inline docs/cloud-native-security/cspm-permissions.asciidoc:4]:
```suggestion
This page lists required privileges for {elastic-sec}'s CSPM features. There are three access levels: read, write, and manage. Each access level and its requirements are described below.
```

File: docs/cloud-native-security/cspm-permissions.asciidoc [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+[discrete]
+== Read
+
+Users with these minimum permissions can view data on the Findings page and the Cloud Posture dashboard.
```
Comment (natasha-moore-elastic) [inline docs/cloud-native-security/cspm-permissions.asciidoc:9]:
```suggestion
Users with these minimum permissions can view data on the **Findings** page and the Cloud Posture dashboard.
```

**Review by natasha-moore-elastic — COMMENTED** [review]
Nice work consolidating all privilege requirements into a single page! Some thoughts about the way the information is presented:

I feel that the current page layout answers the question “What does the read/write/manage privilege allow users to do?” but it could be more helpful to reframe the information so that it answers the question “For each specific action (e.g. enable or disable benchmark rules), what are the minimum privileges a user needs?” On the requirements pages for other features (e.g. [Detections requirements](https://www.elastic.co/guide/en/security/current/detections-permissions-section.html) or [Cases requirements](https://security-docs_bk_6269.docs-preview.app.elstc.co/guide/en/security/master/case-permissions.html)), we do that by using the tabular layout with “Action” as the first column. This focuses more on the different user actions and provides the minimum privileges for each of those actions. I think following the same convention and layout on this page would make the information easier to find and give users a more consistent docs experience.

File: docs/cloud-native-security/cspm-permissions.asciidoc [hunk 3]
```diff
@@ -0,0 +1,61 @@
+[[cspm-required-permissions]]
+= CSPM privilege requirements
```
[the remark this answers is not in the document]
Comment (benironside) [reply docs/cloud-native-security/cspm-permissions.asciidoc:2]: This page is focused only on privileges. Other requirements are listed in the getting started guides

**Review by nastasha-solomon — COMMENTED** [review]
LGTM! If you choose to revisit the content's format, 1+ to @natasha-moore-elastic's suggestion for presenting the information in a table format and with summarized action descriptions.

**Review by natasha-moore-elastic — APPROVED** [review]

**Pull request merged**